src/components/layouts/ui/header/index.ts

```diff
@@ -0,0 +1,8 @@
+import dynamic from 'next/dynamic';
+
+export const Header = Object.assign(
+  dynamic(async () => (await import('./ui/header.main')).default, {
+    ssr: false,
+  }),
+  {}
+);
```

Comment:
빌드 오류 확인 부탁드립니다

Type error: Already included file name '/Users/wonsik/Documents/workspace/codeit/coworkers/src/components/layouts/ui/header/ui/Header.tsx' differs from file name '/Users/wonsik/Documents/workspace/codeit/coworkers/src/components/layouts/ui/header/ui/header.tsx' only in casing.
  The file is in the program because:
    Imported via './ui/Header' from file '/Users/wonsik/Documents/workspace/codeit/coworkers/src/components/layouts/ui/header/index.ts'
    Root file specified for compilation

  1 | import dynamic from 'next/dynamic';
  2 |
> 3 | export const Header = dynamic(() => import('./ui/Header'), { ssr: false });
    |